nautobot/core/tests/integration/test_filters.py

```diff
@@ -152,7 +152,11 @@ def test_input_field_gets_updated(self):
         self.browser.find_by_xpath("//a[@href='#advanced-filter']").click()
         self.assertEqual(self.browser.find_by_name(text_field_name)[2].value, "test new")
         self.assertEqual(self.browser.find_by_name(integer_field_name)[2].value, "1111")
-        self.assertEqual(self.browser.find_by_name(select_field_name)[2].value, "Option B")
+        # CustomSelect Field is a MultiValueCharField, and the only way to get its values is using this approach.
+        # Its values are the  options available on the select field
+        custom_select_field = self.browser.find_by_name(select_field_name)[2].find_by_tag("option")
```

Comment:
Seems like this test should probably be enhanced to also create and test the UI for a multiselect custom field as well?